tests/pipeline/test_extraction.py

```diff
@@ -189,11 +189,11 @@ def test_sextractor_extract_once( decam_datastore, extractor ):
         assert sourcelist.info['SEXBKGND'] == pytest.approx( 179.8, abs=0.1 )
         assert sourcelist.x.min() == pytest.approx( 16.0, abs=0.1 )
         assert sourcelist.x.max() == pytest.approx( 2039.6, abs=0.1 )
-        assert sourcelist.y.min() == pytest.approx( 16.3, abs=0.1 )
+        assert sourcelist.y.min() == pytest.approx( 16.0, abs=0.1 )
         assert sourcelist.y.max() == pytest.approx( 4087.9, abs=0.1 )
         assert sourcelist.apfluxadu(apnum=1)[0].min() == pytest.approx( -656.8731, rel=1e-5 )
         assert sourcelist.apfluxadu(apnum=1)[0].max() == pytest.approx( 2850920.0, rel=1e-5 )
-        assert sourcelist.apfluxadu(apnum=0)[0].min() == pytest.approx( 89.445114, rel=1e-5 )
+        assert sourcelist.apfluxadu(apnum=0)[0].min() == pytest.approx( 84.74182, rel=1e-5 )
```

Comment:
This change (and some of the others, including aperture sizes) will be a change from both RESCALE_WEIGHTS and using is_star in the aperture flux correction (both changes I think should be reverted).